Docs: Timeseries faceted legend filter (#124501)
grafana/grafana: docs/sources/visualizations/panels-visualizations/visualizations/time-series/index.md

{{< docs/shared lookup="visualizations/legend-options-1.md" source="grafana" version="<GRAFANA_VERSION>" leveloffset="+1" >}}

> #### Faceted filter

Reviewer: ```suggestion
#### Series visibility
```

> Toggle the **Faceted filter** switch on to add a filter control next to the legend that lets you narrow the visible series by name or by label.

Reviewer: ```suggestion
Toggle the **Series visibility** switch on to add the control next to or above the legend.

{{< figure src="/media/docs/grafana/panels-visualizations/screenshot-series-visibility-v13.1.png" max-width="400px" alt="Series visibility filter switch" >}}

This lets you narrow the visible series by name or by label.
Use the series visibility filter when a panel renders many series and you want to focus on a subset without editing the query.
```

> Click the filter control to open a popover with two sections:

Reviewer: ```suggestion
After you've toggled the switch on, click the **Series visibility** icon  to open a tooltip with one or two sections depending on your dataset:
```

> - **By name**: Lists each unique series name. Select one or more names to limit the legend to those series.

Reviewer: It doesn't actually change the legend, so this seems a bit incorrect. 
```suggestion
- **By name**: Lists each unique series name. Select one or more names to limit the visualization to those series.
```

- **By labels**: Lists each label key with its values. Select label values to filter series that match.

> The popover also provides:

Reviewer: ```suggestion
The tooltip also provides the following options:
```

- **Select all** and **Deselect all**: Toggle every value in a section.
- **Clear all**: Reset the filter.
- **Pin to sidebar**: Dock the filter alongside the panel so it stays open while you work.

> Use the faceted filter when a panel renders many series and you want to focus on a subset without editing the query.

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

{{< docs/shared lookup="visualizations/legend-options-1.md" source="grafana" version="<GRAFANA_VERSION>" leveloffset="+1" >}}

#### Series visibility

Toggle the **Series visibility** switch on to add the control next to or above the legend.

{{< figure src="/media/docs/grafana/panels-visualizations/screenshot-series-visibility-v13.1.png" max-width="400px" alt="Series visibility filter switch" >}}

This lets you narrow the visible series by name or by label.
Use the series visibility filter when a panel renders many series and you want to focus on a subset without editing the query.

After you've toggled the switch on, click the **Series visibility** icon  to open a tooltip with one or two sections depending on your dataset:

- **By name**: Lists each unique series name. Select one or more names to limit the visualization to those series.
- **By labels**: Lists each label key with its values. Select label values to filter series that match.

The tooltip also provides the following options:

- **Select all** and **Deselect all**: Toggle every value in a section.
- **Clear all**: Reset the filter.

> - **Pin to sidebar**: Dock the filter alongside the panel so it stays open while you work.

Reviewer: ```suggestion
Toggle the **Series visibility** switch on to add the corresponding filter control next to or above the legend.
Use it to narrow the visible series by name or by label without editing the query, which is useful when a panel renders many series and you want to focus on a subset.

After toggling the switch on, click the **Series visibility** icon to open a tooltip.
Depending on your dataset, you can filter: 

- **By name**: Lists each unique series name. Select one or more names to limit the visualization to those series.
- **By labels**: Lists each label key with its values. Select label values to filter series that match.
- 

For more information, refer to the [Configure legend documentation] (https://grafana.com/docs/grafana/<GRAFANA_VERSION>/visualizations/panels-visualizations/configure-legend/#series-visibility).
```

After you've toggled the switch on, click the **Series visibility** icon to open a tooltip with one or two sections depending on your dataset:

- **By name**: Lists each unique series name. Select one or more names to limit the visualization to those series.
- **By labels**: Lists each label key with its values. Select label values to filter series that match.

The tooltip also provides the following options:

- **Select all** and **Deselect all**: Toggle every value in a section.
- **Clear all**: Reset the filter.
  Toggle the **Series visibility** switch on to add the corresponding filter control next to or above the legend.
  Use it to narrow the visible series by name or by label without editing the query, which is useful when a panel renders many series and you want to focus on a subset.

After toggling the switch on, click the **Series visibility** icon to open a tooltip.
Depending on your dataset, you can filter:

- **By name**: Lists each unique series name. Select one or more names to limit the visualization to those series.
- **By labels**: Lists each label key with its values. Select label values to filter series that match.

> -

Reviewer: ```suggestion
```
